sdk/servicebus/microsoft-azure-servicebus/README.md

```diff
@@ -59,7 +59,7 @@ If you send messages from a client any other than the Java client itself in any
 ## FAQ
 
 ### Where is the API document?
-Click [here](https://docs.microsoft.com/en-us/java/api/overview/azure/servicebus/clientlibrary).
+Click [here](https://docs.microsoft.com/java/api/overview/azure/servicebus/clientlibrary).
```

Comment:
This link gives a 404, is it still under development?

Reply:
It's the old track 1 library... I don't where it should even link to. lol. I'll find a home for it.